unitTests/magicbox/SuggestionsManagerTest.ts

```diff
@@ -79,6 +74,22 @@ export function SuggestionsManagerTest() {
       expect(selectedWithKeyboard).toBe(suggestion.el);
     });
 
+    it('returns the correct selected element with keyboard on move left', () => {
+      suggestionManager.moveLeft();
+      const selectedWithKeyboard = suggestionManager.selectAndReturnKeyboardFocusedElement();
+      expect($$(selectedWithKeyboard).hasClass(selectedClass)).toBe(true);
+      expect($$(selectedWithKeyboard).getAttribute('aria-selected')).toBe('true');
+      expect(selectedWithKeyboard).toBe(suggestion.el);
+    });
+
+    it('returns the correct selected element with keyboard on move right', () => {
+      suggestionManager.moveRight();
+      const selectedWithKeyboard = suggestionManager.selectAndReturnKeyboardFocusedElement();
+      expect($$(selectedWithKeyboard).hasClass(selectedClass)).toBe(true);
+      expect($$(selectedWithKeyboard).getAttribute('aria-selected')).toBe('true');
+      expect(selectedWithKeyboard).toBe(suggestion.el);
+    });
+
```

Comment:
There might be more cases when going left/right, whether you're at the edges or not for example. That would test the code you wrote for those "edge" cases. 😛

Reply:
I totally agree but here's I seem to have a problem with my template. 

Only 1 element gets rendered. 
I think the problem is in `setupQuerySuggestPreview` the way I mock my component template probably but I'm not really sure why.

I don't think I could be able to finish this in like an hour so I'll leave it like this for now.

I could open a Jira to add a unit test or we could have a functional test that could also validate that the CSS is applied

Olivier asked me to documents the more I could on the epic. I'll add it there.